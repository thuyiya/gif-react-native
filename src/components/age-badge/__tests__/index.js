/* eslint-disable no-undef */
import React from "react";
// import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";
import { AgeBadge } from "_components";

describe("AgeBadge", () => {
  it("renders correctly react-test-renderer", () => {
    const renderer = new ShallowRenderer();
    renderer.render(<AgeBadge text={"16+"} />);
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
