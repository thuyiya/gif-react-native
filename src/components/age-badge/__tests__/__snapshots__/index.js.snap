// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgeBadge renders correctly react-test-renderer 1`] = `
<View
  accessible={true}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#5e5d5d",
        "borderRadius": 32,
        "height": 64,
        "justifyContent": "center",
        "width": 64,
      },
      undefined,
    ]
  }
>
  <Text
    accessibilityLabel="AGE"
    style={
      Array [
        Object {
          "color": "#FFFFFF",
          "fontWeight": "600",
        },
        Object {
          "fontSize": 26,
        },
      ]
    }
  >
    16+
  </Text>
</View>
`;
